### ISSUE TITLE
suggestions: 1. Size slider for window/icon sizes, 2. keep taskbar visible when auto-hide is enabled

### ISSUE THREAD

**Issue by johnysandels** [tjackenpacken/taskbar-groups#133]
I imagine the second thing could be done by grabbing focus while the taskbar group is open

**Comment by PikeNote** [comment]
To clarify, slider to adjust the icon sizes when it pops up and keep the taskbar visible when the program does launch.

**Comment by johnysandels** [comment]
Yes they're two seperate suggestions, just wrote them both in the title so it would be easier to find/search
